dotcom-rendering/src/web/components/App.tsx

```diff
@@ -1167,20 +1172,22 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				</Portal>
 			)}
 			<Portal rootId="slot-body-end">
-				<SlotBodyEnd
-					isSignedIn={isSignedIn}
-					countryCode={countryCode}
-					contentType={CAPI.contentType}
-					sectionName={CAPI.sectionName}
-					shouldHideReaderRevenue={CAPI.shouldHideReaderRevenue}
-					isMinuteArticle={CAPI.pageType.isMinuteArticle}
-					isPaidContent={CAPI.pageType.isPaidContent}
-					tags={CAPI.tags}
-					contributionsServiceUrl={CAPI.contributionsServiceUrl}
-					brazeMessages={brazeMessages}
-					idApiUrl={CAPI.config.idApiUrl}
-					stage={CAPI.stage}
-				/>
+				{articleCountIsReady && (
```

Comment:
Checking the rest of `App` it looks like the other places where we're conditionally rendering `Portal`s the check is outside the `Portal`, eg:

```
			{CAPI.matchUrl && (
				<Portal rootId="match-nav">
					<GetMatchNav matchUrl={CAPI.matchUrl} />
				</Portal>
			)}
```

I don't _think_ it will matter too much if we render an empty portal but having one less call to `ReactDOM.createPortal` is nice

Reply:
Yeah looks good I'll push up that change now!